jsontableschema_sql/storage.py

```diff
@@ -81,7 +82,7 @@ def check(self, table):
 
         return existence
 
-    def create(self, table, schema):
+    def create(self, table, schema, indexes_fields = []):
```

Comment:
After you've proposed to move default arguments to signature in GT (also I've done it in tabulator) I do opposite - all default in signatures and for mutable one I use `copy` on assignment to clear it.